xlsxwriter.class.php

```diff
@@ -363,7 +363,7 @@ protected function writeCell(XLSXWriter_BuffererWriter &$file, $row_number, $col
 		if (!is_scalar($value) || $value==='') { //objects, array, empty
 			$file->write('<c r="'.$cell_name.'" s="'.$cell_style_idx.'"/>');
 		} elseif (is_string($value) && $value{0}=='='){
-			$file->write('<c r="'.$cell_name.'" s="'.$cell_style_idx.'" t="s"><f>'.self::xmlspecialchars($value).'</f></c>');
+			$file->write('<c r="'.$cell_name.'" s="'.$cell_style_idx.'" t="s"><f>'.self::xmlspecialchars(str_replace('=', '', $value)).'</f></c>');
```

Comment:
Could this better be `substr($value, 1)`? Or is it impossible for the value to contain another `=`?

Reply:
If it is definitive a formula, the first letter from left is az equal sign so ` ltrim($value, '=')  `  is the best solution

Reply:
You are right.